### ISSUE TITLE
#T03  Landing page

### ISSUE THREAD

**Issue by alexeykuz-sys** [Code-Institute-Community/ci-hackathon-app#57]
Landing page - created bootstrap accordion and carrousel blocks on landing page. positioned next to each other and made sure they are responsive. used orange color and black font to improve color contrast 
![Screen Shot 2020-10-21 at 12 50 45](https://user-images.githubusercontent.com/64777928/96715922-3205d100-139c-11eb-93c1-d1d647fe2c0f.png) 
Also check with html and css validators and got all good.The page lacks content, which will be added later. Question: how can i add content?
![Screen Shot 2020-10-21 at 12 53 10](https://user-images.githubusercontent.com/64777928/96716050-6bd6d780-139c-11eb-99e7-61fdd4dadb04.png)

**Comment by TravelTimN** [comment]
@alexeykuz-sys thanks for opening a separate issue for this one! 👍 
Make sure to just tag yourself on the comment, but I see your original message on #53 so I'll manually add you now.
Thanks! 😁

**Comment by TravelTimN** [comment]
Updating #ID to `#T03` for `T`emplate user story
